config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,23 @@
+Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
+Rack::Attack.throttled_response = ->(env) { [429, {}, [ActionView::Base.new.render(file: 'public/429.html')]] }
+
+ActiveSupport::Notifications.subscribe('rack.attack') do |name, start, finish, request_id, req|
+  Airbrake.notify Exception.new(message: "#{req.ip} has been rate limited for url #{req.url}", data: req.env['rack.attack.throttle_data'])
```

Comment:
Notifies airbrake with a message and the throttle data, which looks like:

```
{ count: 6, limit: 5, period: 60}
```
Ie, this person attempted to hit this endpoint 6 times in the last 60 seconds, which is greater than the rate limit of 5.